blocks/init/src/blocks/components/hamburger/hamburger-style.scss

```diff
@@ -0,0 +1,78 @@
+/* stylelint-disable max-nesting-depth */
+
+$hamburger: (
+  colors: (
+    line: #2C2C2C,
```

Comment:
use only colors that we are using here https://github.com/infinum/eightshift-frontend-libs/blob/develop/styles/scss/_colors.scss

or add a nother one to the _colors list